tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import (
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/cloudprovider"
+	"testing"
+	"time"
+)
+
+func TestE2E(t *testing.T) {
```

Comment:
An e2e should be specific. It needs to explain what it is testing for. The name should also reflect that. How are you going to kick this test off from the command line?